addon/helpers/number.js

```diff
@@ -3,6 +3,15 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([value = 0]) {
+    let num = Number(value);
+    if (`${num}` === 'NaN') {
```

Comment:
Is this what we want to do here? Getting too fancy with coercion is where many of the famous JavaScript `WAT`s come from. I think we should just let `NaN` be `NaN` either that, or throw an exception.

> aside:  you can use the global function `isNaN` to check if a value is `NaN`